tests/unit/synapseclient/unit_test_tables.py

```diff
@@ -119,6 +121,106 @@ def test_cast_values__list_type():
     ]
 
 
+def test_convert_df_date_cols_to_datetime_empty_df():
+    test_df = pd.DataFrame()
+    date_columns = []
+    test_df_output = _convert_df_date_cols_to_datetime(test_df, date_columns)
+    assert test_df_output.empty
+
+
+def test_convert_df_date_cols_to_datetime_invalid_epoch_time():
+    test_df = pd.DataFrame(
```

Comment:
Explore using a `pytest.fixture` for this.

You can create one invalid dataframe that can be reused across these two tests. The first error will be the invalid column headers then it will test if the values in the columns are valid.

Reply:
Yes, that's a good idea. I have added fixture so that the dataframe can be used for multiple tests